hugegraph-core/src/main/java/org/apache/hugegraph/util/Consumers.java

```diff
@@ -43,55 +43,55 @@ public final class Consumers<V> {
     public static final int THREADS = 4 + CoreOptions.CPUS / 4;
     public static final int QUEUE_WORKER_SIZE = 1000;
     public static final long CONSUMER_WAKE_PERIOD = 1;
+    private static final Object QUEUE_END = new VWrapper(null);
```

Comment:
can we remove VWrapper class and just use `new Object()` instead

Reply:
`QUEUE_END` will be put at the end of `BlockingQueue<VWrapper<V>> queue`, so it will be cast to `VWrapper<V>`.

If we just use `new Object()` instead, we will get the exception below.

<img width="709" alt="image" src="https://github.com/apache/incubator-hugegraph/assets/77946882/3ea7d102-bfde-4543-9531-c384e7ef5cea">

Reply:
get it, can we remove VWrapper class and keep `BlockingQueue<V> queue`, then try to define `V QUEUE_END = (V) new Object();`